internal/util/operator-registry/package_manifest.go

```diff
@@ -17,12 +17,11 @@ package registry
 import (
 	"fmt"
 
+	"github.com/operator-framework/operator-registry/pkg/registry"
 	"github.com/pkg/errors"
-
-	olmregistry "github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry"
 )
 
-func ValidatePackageManifest(pm *olmregistry.PackageManifest) error {
+func ValidatePackageManifest(pm *registry.PackageManifest) error {
```

Comment:
Are we duplicating the impl of the object defined in OLM? 
Should not be better we use the object from OLM in order to keep the impl centralized there? If something in this struct changed then will not be required replicated the same impl for here?

Reply:
This is covered in the PR description I think:

> OLM's PackageManifest type is used for display purposes, while operator-registry's is used for (un)marshalling package manifests.

Reply:
WDYT about we add comments in the code in order to make clear this diff purposes? Something that could help to avoid the same happened again( i mean the olmregistry.PackageManifest be used instead of registry.PackageManifest)